File: src/main/java/com/sfh/pokeRogueBot/model/decisions/PossibleAttackMovesForDoubleFight.java
package com.sfh.pokeRogueBot.model.decisions;

import lombok.Data;


@Data
public class PossibleAttackMovesForDoubleFight {

    private final ChosenAttackMove chosenFinisher1;
    private final ChosenAttackMove chosenFinisher2;
    private final ChosenAttackMove maxDmgMove1;
    private final ChosenAttackMove maxDmgMove2;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/decisions/PossibleAttackMovesForDoubleFight.kt
package com.sfh.pokeRogueBot.model.decisions

data class PossibleAttackMovesForDoubleFight(
    val chosenFinisher1: ChosenAttackMove?,
    val chosenFinisher2: ChosenAttackMove?,
    val maxDmgMove1: ChosenAttackMove?,
    val maxDmgMove2: ChosenAttackMove?
)